### DISCUSSION TITLE
web-ui 코드 테스트

### DISCUSSION THREAD

**Discussion by yujung7768903** [lablup/contribution-academy-2021#108] [Q&A] [answered]
web-ui 관련 이슈를 해결하면서 수정한 코드들을 테스트해보고 싶은데, 잘 되지 않아 질문 올립니다.
web-ui 컴포넌트 readme파일에서 다음부분을 참고했습니다.
<img width="639" alt="Screen Shot 2021-10-04 at 2 06 28 PM" src="https://user-images.githubusercontent.com/68562176/135797228-2eb5c13f-7e5f-41a0-98fc-6406fb115e90.png">

npm run server:d로 서버 돌린 후에 Lacalhost:8090으로 들어가면 사이트가 뜨긴뜨는데 수정한 부분이 반영되지 않습니다.
Npm run build:d 실행 후에 localhost:9081/로 들어가면 사이트가 뜨지만 아래와 같이 에러가 뜹니다.
<img width="1433" alt="Screen Shot 2021-10-04 at 2 14 30 PM" src="https://user-images.githubusercontent.com/68562176/135797736-a8e9ea1d-3e38-4daf-a1b1-8a4a3ed37292.png">
그리고 9081로 접속했을 때는 개발자 도구 source탭을 확인해봤을 때 8090과 다르게 dist/components 폴더가 존재하지 않았습니다.
<img width="952" alt="Screen Shot 2021-10-04 at 2 16 33 PM" src="https://user-images.githubusercontent.com/68562176/135797870-f0e247da-da68-49a8-a689-b639046fda66.png">

수정한 부분을 테스트해보려면 어떻게 해야하나요?

**Comment by lizable** [comment, marked as answer]
@yujung7768903 님 혹시 해당 branch에 main branch 최근 commit 땡겨오셨는지요? 그리고 그 후에도 잘 안될 경우에는 node_modules 를 과감히 삭제하시고 `npm i` 로 노드패키지 변경내역에 맞는 라이브러리들을 다시 받아보시기를 추천합니다.
